website/docs/targeting/targeting-rule/targeting-rule-overview.mdx

```diff
@@ -18,30 +20,30 @@ The Targeting Rules are in an **OR** relationship, meaning that the Targeting Ru
 
 ## How to add a Targeting Rule?
 
-You can add a Targeting Rule to a feature flag on the Dashboard by clicking on the `+IF` ("Add Targeting Rule") button. Add more Targeting Rules by clicking on the `+OR` button at the bottom of the Targeting Rule.
+You can add a Targeting Rule to a feature flag on the Dashboard by clicking on the **+IF** ("Add targeting rule") button, then more Targeting Rules by clicking on the **+OR** button at the bottom of the last Targeting Rule.
 
-![Add Targeting Rule](/assets/targeting/targeting-rule/add-rule.png)
+<img alt="Add targeting rule" src="/docs/assets/targeting/targeting-rule/add-targeting-rule_192dpi.png" width="700" height="525" decoding="async" loading="lazy" />
 
 ## How does it work? - Anatomy of a Targeting Rule
 
-A Targeting Rule consists of an **IF part** and a **THEN part**.
+A Targeting Rule consists of an _IF part_ and a _THEN part_.
 
-![Targeting Rule anatomy](/assets/targeting/targeting-rule/targeting-rule.png)
+<img alt="Targeting rule anatomy" src="/docs/assets/targeting/targeting-rule/targeting-rule-anatomy_192dpi.png" width="1084" height="433" decoding="async" loading="lazy" />
 
 ### IF part
 
-The *IF part* contains the conditions, which are logical expressions that evaluate to true or false.
+The _IF part_ contains the conditions, which are logical expressions that evaluate to true or false.
 
 The conditions are in an **AND** relationship, meaning that all of them must evaluate to true for the Targeting Rule to match.
 
-The conditions can be added to the Targeting Rule on the Dashboard. There are three types of conditions:
+The conditions can be added to the Targeting Rule on the Dashboard, using the **+AND** ("Add AND condition") button. There are three types of conditions:
 - [User Condition](../user-condition) - A condition that is based on some property of the user.
 - [Flag Condition (Prerequisite)](../flag-condition) - A condition that is based on the value of another feature flag.
 - [Segment Condition](../segment-condition) - A condition that is based on a segment.
 
 ### THEN part
 
-The *THEN part* contains either a value or Percentage Options that will be used to determine the result of the feature flag when the Targeting Rule matches.
+The _THEN part_ contains either a value or Percentage Options that will be used to determine the result of the feature flag when the Targeting Rule matches.
```

Comment:
```suggestion
The _THEN part_ determines the result of the feature flag when the Targeting Rule matches. It can be either a simple value or Percentage Options.
```